src/OmniSharp.Abstractions/Models/v2/CodeStructure/CodeElementKinds.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace OmniSharp.Models.V2.CodeStructure
+{
+    public static class CodeElementKinds
+    {
+        // types
+        public static readonly string Class = nameof(Class).ToLowerInvariant();
+        public static readonly string Delegate = nameof(Delegate).ToLowerInvariant();
+        public static readonly string Enum = nameof(Enum).ToLowerInvariant();
+        public static readonly string Interface = nameof(Interface).ToLowerInvariant();
+        public static readonly string Struct = nameof(Struct).ToLowerInvariant();
+
+        // members
```

Comment:
should this include `IndexerDeclaration` too?

Reply:
Yup. I just noticed that. Fixed.